app/views/admin_mailer/abuse_report.html.erb

```diff
@@ -4,6 +4,9 @@
   
   Email of the person who reported the abuse, if given: <%= style_email(@email, @email) %><br>
   URL of the page the abuse is on: <%= style_link(@url, @url) %><br>
+  
+  Title of the work: <%= @title.html_safe %><br/>
+  Author(s) of the work: <%= @author.html_safe %><br/> 
```

Comment:
Drive-by front end nitpick: to be less text-centric, we're using "creator" instead of "author" in any interface/email/etc text now.